server/config/environment/index.js

```diff
@@ -25,6 +25,11 @@ var all = {
   // Postmark Token
   token: process.env.POSTMARK_TOKEN,
 
+  everyAction: {
+    appName: process.env.EVERYACTION_APP_NAME,
```

Comment:
@rvillanueva We have an API key generated; any tips on how to get these two variables into the prod environment for it to authenticate to the EveryAction API?

Reply:
Pass them to me on Slack or something secure and I can add it to our secrets store!